code/modules/shuttle/shuttle_rotate.dm

```diff
@@ -80,6 +80,10 @@ If ever any of these procs are useful for non-shuttles, rename it to proc/rotate
 	params &= ~ROTATE_OFFSET
 	return ..()
 
+/obj/structure/reflector/shuttleRotate(rotation, params)
+	. = ..()
+	setAngle(SIMPLIFY_DEGREES(rotation_angle += rotation))
```

Comment:
```suggestion
	setAngle(SIMPLIFY_DEGREES(rotation_angle + rotation))
``` 
might only need to be this? I could be wrong

Reply:
setAngle already sets rotation_angle, and I need SIMPLIFY_DEGREES to check for degrees greater than 360. Rotation_angle + rotation is just getting the new angle we want to set.

Reply:
Yep, in that case you'll only need + since += adds and then sets the var to it.